### ISSUE TITLE
Socket seems to go stale

### ISSUE THREAD

**Issue by kaihendry** [kilianc/rtail#91]
Do you think you could add something in the UI, to show the socket is open and active?

For some reason or another my client seems to go stale and I'm assuming it's lost connection.

**Comment by kilianc** [comment]
Yes this is on our radar see: https://github.com/kilianc/rtail/issues/77
